.github/workflows/pytest.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: run tests
       run: |
         pip install -U pip
-        pip install --upgrade coveralls setuptools setuptools_scm pep517
+        pip install --upgrade coveralls setuptools setuptools_scm pep517 snoop
```

Comment:
Move `snoop` to `setup.cfg` as a test requirement, similar to pytest, so the line below will install it.